libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -271,6 +271,15 @@ class MapboxNavigation(
      */
     val historyRecorder = MapboxHistoryRecorder(navigationOptions, logger)
 
+    /**
+     * Navigation native experimental API. Do not store a link to [Experimental] object, use it with
```

Comment:
```suggestion
    /**
     * **THIS IS AN EXPERIMENTAL API, DO NOT USE IN A PRODUCTION ENVIRONMENT.**
     *
     * Navigation native experimental API. Do not store a link to [Experimental] object, use it with
```

or something along these lines to send an even clearer message :)